plugins/terminal.koplugin/profile

```diff
@@ -7,6 +7,11 @@
 
 # shellcheck source=/dev/null
 
+if [ -x "${SHELL}" ] && [ "${SHELL}" != "$0" ]; then
+    exec "${SHELL}" "$@"
+    exit 0
```

Comment:
I included this somewhat on autopilot, but since exec replaces the current shell the only reason we'd ever reach this is a failure, in which case it should probably just carry on (and if not it should be > 0).

```suggestion
```